app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java

```diff
@@ -202,15 +202,11 @@ public String getPageContents(Context applicationContext) {
     private String getTemplatizedCreatedDate() {
         if (dateCreated != null) {
             if (UploadableFile.DateTimeWithSource.EXIF_SOURCE.equals(dateCreatedSource)) {
-                return String.format(Locale.ENGLISH, TEMPLATE_DATE_ACC_TO_EXIF, DateUtil.getDateStringWithSkeletonPattern(dateCreated, "yyyy-MM-dd")) + "\n";
+                java.text.SimpleDateFormat dateformat = new java.text.SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Please rename the variable to `dateFormat`, thanks! 🙂

Reply:
@PavelAplevich  Please rename dateformat to dateFormat :)

Reply:
@ashishkumar468 All done already, see last commit